qcodes/loops.py

```diff
@@ -263,7 +263,7 @@ def with_bg_task(self, task, end_task=None, min_delay=1):
             end_task: A callable object with no parameters. This object will be
                 invoked at the end  of the measurement loop.
 
-            min_delay (default 1): The minimum number of seconds to wait
+            min_delay (default 0.01): The minimum number of seconds to wait
```

Comment:
That is indeed a problem with matplotlib, yes we should definitely add that to avoid nasty surprises!